tempesta_fw/cfg.c

```diff
@@ -848,6 +848,10 @@ spec_cleanup(TfwCfgSpec specs[])
 			spec->cleanup(spec);
 		}
 		spec->call_counter = 0;
+
+		if (spec->handler == &tfw_cfg_handle_children) {
```

Comment:
Looks like solving some issue. It would be nice to have that in a separate commit.